### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by pgrunewald**
As discussed on IRC. Drop Python 2.6 and allow pypy3 to fail.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@ matrix:
   include:
     - python: 3.5
       env: TOXENV=py35
+  allow_failures:
+  - python: 'pypy3'
```
Comment (gforcada) [inline .travis.yml]: @pgrunewald seems that this is not the syntax travis expects, I guess, that you should put ``TOXENV=pypy3`` here

**Comment by gforcada** [comment]
👍 good job, there's a small fix to mark pypy3 safe though, see my inline comment

**Comment by pgrunewald** [comment]
Thanks for the review!

File: .travis.yml [hunk 2]
```diff
@@ -4,8 +4,9 @@ matrix:
   include:
     - python: 3.5
       env: TOXENV=py35
+  allow_failures:
+  - python: 'TOXENV=pypy3'
```
Comment (gforcada) [inline .travis.yml]: @pgrunewald finally I found the correct syntax!

```
allow_failures:
  - python:
    env: 'TOXENV=pypy3'
```

**Comment by pgrunewald** [comment]
This should hopefully do it. We better squash those commits into a single one to spare the embarrassment in the history. :)

**Comment by gforcada** [comment]
*there's no need to run jenkins jobs as the only changes are on .travis.yml*

**Pull request merged**